QUICK_START_GUIDE.md

```diff
@@ -26,385 +26,381 @@ Current version of OTX was tested under following environments
 
    ```bash
    # Create virtual env.
-   python -m venv .venv
+   python -m venv .otx
 
    # Activate virtual env.
-   source .venv/bin/activate
+   source .otx/bin/activate
    ```
 
 1. Install prerequisite dependencies with:
 
-   Install pytorch according to your system environment.
-   Refer to the [official inatllation guide](https://pytorch.org/get-started/previous-versions/)
+   Install PyTorch according to your system environment.
+   Refer to the [official installation guide](https://pytorch.org/get-started/previous-versions/)
 
-   > **_Important note:_** Currently, only torch==1.3.1 was fully validated. torch==2.x will be supported soon. (Earlier versions are not supported due to security issues)
+   > **_Important note:_** Currently, only torch==1.13.1 was fully validated. torch==2.x will be supported soon. (Earlier versions are not supported due to security issues)
 
    ```bash
-   # Example install command for torch==1.13.1+cu116
-   pip install torch==1.13.1 torchvision==0.14.1 --extra-index-url https://download.pytorch.org/whl/cu116
+   # Example install command for torch==1.13.1+cu117
+   pip install torch==1.13.1 torchvision==0.14.1 --extra-index-url https://download.pytorch.org/whl/cu117
    ```
 
-   (Optional) You may also want to use Jupyter notebooks or OTX CLI tools:
-
-   ```bash
-   pip install notebook; cd ote_cli/notebooks/; jupyter notebook
-   ```
-
-   > **_Important note:_** You should confirm that the Python version that installed on your machine should be 3.8.X. For the future release of OTX will support wide range of the Python version.
-
 1. Then, install OTX package
 
    ```bash
-   # Install from local source in development mode
+   # Install from a local source in development mode
    pip install -e .[full]
 
    # Or, you can install from PyPI
    pip install otx  # Will be available from otx==1.0.0
    ```
 
-1. Once the package is installed to the virtual environment, you can use the
-   `otx` command line interface to perform various commands for templates related to the chosen task type, described in [OTX CLI commands](#otx-cli-commands) on that virutal environment.
+1. Once the package is installed in the virtual environment, you can use full
+   `otx` command line functionality as described in [OTX CLI commands](#otx-cli-commands).
 
 ### Install OTX for developers
 
-#### Run tests
-
-```bash
-# Install development prerequisite
-python -m pip install -r requirements/dev.txt
-
-# Run tests
-tox -e pre-merge
-tox -e pre-merge
-```
-
 #### Development environment
 
 ```bash
-# Create development environment
-# -- need to fix '38' in tox.ini if other python version needed
-tox devenv -e pre-merge venv/otx
+# Create a development environment
+# -- need to fix '38' in tox.ini if another python version needed
+tox devenv venv/otx -e pre-merge
```

Comment:
Nit: `python -m pip install -r requirements/dev.txt` is actually omitted here, but users could just hit `pip install otx`.